api/src/pcapi/core/search/__init__.py

```diff
@@ -381,6 +385,42 @@ def index_offers_of_venues_in_queue() -> None:
         logger.exception("Could not index offers of venues from queue")
 
 
+def get_base_query_for_offer_indexation() -> BaseQuery:
+    return (
+        offers_models.Offer.query.options(
+            sa.orm.joinedload(offers_models.Offer.venue).joinedload(offerers_models.Venue.managingOfferer)
+        )
+        .options(sa.orm.joinedload(offers_models.Offer.criteria))
+        .options(sa.orm.joinedload(offers_models.Offer.mediations))
+        .options(sa.orm.joinedload(offers_models.Offer.product))
+        .options(sa.orm.joinedload(offers_models.Offer.stocks))
+    )
+
+
+def get_base_query_for_last_30_days_bookings() -> BaseQuery:
+    return (
+        sa.select(
+            offers_models.Offer.id.label("offer_id"), sa.func.count(bookings_models.Booking.id).label("bookings_number")
+        )
+        .select_from(offers_models.Offer)
+        .join(offers_models.Stock, offers_models.Stock.offerId == offers_models.Offer.id)
+        .join(bookings_models.Booking, bookings_models.Booking.stockId == offers_models.Stock.id)
+        .where(
+            bookings_models.Booking.dateCreated >= datetime.datetime.utcnow() - datetime.timedelta(days=30),
+            sa.or_(
+                bookings_models.Booking.status.notin_(
+                    (
+                        bookings_models.BookingStatus.PENDING,
+                        bookings_models.BookingStatus.CANCELLED,
+                    )
+                ),
+                bookings_models.Booking.status.is_(None),
+            ),
```

Comment:
Booking.status est `NOT NULL`. Et `BookingStatus.PENDING` n'est plus utilisé (et en train d'être supprimé dans une pull request en cours). Donc ça peut donner ça : 

```suggestion
            bookings_models.Booking.status != bookings_models.BookingStatus.CANCELLED,
```